README.md

```diff
@@ -0,0 +1,10 @@
+# java
+Work In Progress
+
+# Update client
+
+to update the client clone `gen` repo and run this command at the root of the client repo:
```

Comment:
clone _the_ `gen` repo.

(personally, I would just add a simple bash script to this repo that does this, but it could be a follow-on)